### PR TITLE
pkg/syncmap: add unit test for GetAsAtomicInt

### PR DESCRIPTION
**Pull request by q384566678**
Signed-off-by: Zhou Hao <zhouhao@cn.fujitsu.com>

### CODE REVIEW

File: pkg/syncmap/syncmap_test.go [hunk 1]
```diff
@@ -131,3 +133,12 @@ func (suite *SyncMapUtilSuite) TestGetAsTime(c *check.C) {
...
 	c.Check(result, check.DeepEquals, expected)
 }
+
+func (suite *SyncMapUtilSuite) TestGetAsAtomicInt(c *check.C) {
+	expected := atomiccount.NewAtomicInt(10)
+	mmap := NewSyncMap()
+	mmap.Add("aaa", expected)
+
+	result, _ := mmap.GetAsAtomicInt("aaa")
+	c.Check(result, check.DeepEquals, expected)
```
Comment (allencloud) [inline pkg/syncmap/syncmap_test.go]: You could add code here to check the non-existent one, then check the non-nil error:
```
result, err := mmap.GetAsAtomicInt("nonexist")
```

Comment (q384566678) [reply pkg/syncmap/syncmap_test.go]: updated, PTAL

**Comment by allencloud** [comment]
CI failed, PTAL:
```
./hack/golangci-lint.sh
Begin to check gocode with golangci-lint locally
Detected that golangci-lint has already been installed. Start linting...
pkg/syncmap/syncmap_test.go:145:2: SA4006: this value of `result` is never used (staticcheck)
	result, err := mmap.GetAsAtomicInt("nonexist")
	^
Makefile:160: recipe for target 'golangci-lint' failed
make: *** [golangci-lint] Error 1
```

File: pkg/syncmap/syncmap_test.go [hunk 2]
```diff
@@ -131,3 +133,15 @@ func (suite *SyncMapUtilSuite) TestGetAsTime(c *check.C) {
...
+
+	result, err := mmap.GetAsAtomicInt("nonexist")
+	c.Check(err, check.NotNil)
+}
```
Comment (allencloud) [inline pkg/syncmap/syncmap_test.go]: Please add `c.Check(result, check.IsNil)`

Comment (q384566678) [reply pkg/syncmap/syncmap_test.go]: updated, thanks.

**Comment by allencloud** [comment]
LGTM

**Pull request merged**